vmm/src/vm.rs

```diff
@@ -491,6 +493,22 @@ impl UserIoapicIrq {
     }
 }
 
+pub fn get_win_size() -> (u16, u16) {
+    struct WS {
```

Comment:
You need to `#repr(C)` this.

Reply:
Might be simpler to use a "tuple struct"

Reply:
How to return a 'tuple' from 'tuple struct'?
I used  `#[repr(C)]` before the struct definition.

Reply:
> I have used #[repr(C)] before the struct definition.

I don't see this.

Reply:
Could you please check now?